action.yml

```diff
@@ -32,6 +32,8 @@ inputs:
     description: 'Explicitly set a release name. Defaults to empty which will cause the release to take the tag as name on GitHub.'
   body:
     description: 'Content of the release text. Empty by default.'
+  target_commit:
+    description: 'Specifies the commitish value that determines where the Git tag is created from. Can be any branch or commit SHA. Unused if the Git tag already exists. Default: the repository"s default branch (usually `master`).'
```

Comment:
I think you should warn if the commit already exists.

Reply:
Sorry, I didn't get you. Why should there be a warning if the commit exists?

Reply:
I think the action should emit a warning using `core.warning()` if the specified `target_commit` is not getting used. It will hopefully make users less confused.

Reply:
I misunderstood your earlier comment but I've added the warning now. However, the get tags API is returning a 404 for me and I'm not sure why that's the case. Even the gh CLI is returning 404 when I tried it locally on a sample repo that had a tag.

Reply:
I also found the API to be a bit tricky in places. Sadly I can't really advise you on it, it usually involves a fair bit of trial and error for me.

Reply:
Hey @svenstaro. Sorry for the delay but I fixed this. I used the get refs API and tested it, works as expected now.